<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string>enable</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButton">
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabsClosable">
      <bool>true</bool>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="Button">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>PushButton</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>QPushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_normal">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>normal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_default">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>default</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_warning">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>warning</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_flat">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="window"/>
         </property>
         <property name="iconSize">
          <size>
           <width>43</width>
           <height>43</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_toggle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>toggle</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>SwitchButton</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
     <widget class="QWidget" name="tab_2">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <attribute name="title">
       <string>ComboBox</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>测试测试测试测试测试测试测试1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>测试测试测试测试测试测试测试1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>测试测试测试测试测试测试测试3</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bth_hasMenu">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>button with menu</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>edit</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans CJK SC Light'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="placeholderText">
          <string>QTextEdit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="text">
          <string>QLineEdit</string>
         </property>
         <property name="placeholderText">
          <string>QLineEdit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="styleSheet">
          <string notr="true">QLineEdit{
lineedit-password-character: 9678;
}</string>
         </property>
         <property name="text">
          <string>QLineEdit Passwd</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>QLineEdit Passwd</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="plainText">
          <string>QPlainTextEdit</string>
         </property>
         <property name="placeholderText">
          <string>QPlainTextEdit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QKeySequenceEdit" name="keySequenceEdit"/>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit"/>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit"/>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>ScrollBar</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>296</width>
            <height>834</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QToolBox" name="toolBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>666</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>666</height>
              </size>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="page_3">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>278</width>
                <height>598</height>
               </rect>
              </property>
              <attribute name="label">
               <string>Page 1</string>
              </attribute>
             </widget>
             <widget class="QWidget" name="page_4">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>98</width>
                <height>28</height>
               </rect>
              </property>
              <attribute name="label">
               <string>Page 2</string>
              </attribute>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <widget class="QStackedWidget" name="stackedWidget">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>60</y>
                <width>120</width>
                <height>80</height>
               </rect>
              </property>
              <widget class="QWidget" name="page"/>
              <widget class="QWidget" name="page_2"/>
             </widget>
             <widget class="QGroupBox" name="groupBox">
              <property name="geometry">
               <rect>
                <x>140</x>
                <y>0</y>
                <width>120</width>
                <height>80</height>
               </rect>
              </property>
              <property name="title">
               <string>GroupBox</string>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QScrollArea" name="scrollArea_2">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_2">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>276</width>
                <height>68</height>
               </rect>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QTabWidget" name="tabWidget_2">
             <widget class="QWidget" name="tab_5">
              <attribute name="title">
               <string>Tab 1</string>
              </attribute>
             </widget>
             <widget class="QWidget" name="tab_6">
              <attribute name="title">
               <string>Tab 2</string>
              </attribute>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>SearchBox</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8"/>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>ProgressBar</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>QListView</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="KiranSidebarWidget" name="listWidget">
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_10">
      <attribute name="title">
       <string>KiranIconLineEdit</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="KiranIconLineEdit" name="iconLineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_imageSelector">
      <attribute name="title">
       <string>KiranImageSelector</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_11"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KiranSidebarWidget</class>
   <extends>QListWidget</extends>
   <header location="global">kiran-sidebar-widget.h</header>
  </customwidget>
  <customwidget>
   <class>KiranIconLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">kiran-icon-line-edit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
